ContextApp/src/main/java/com/github/migi_1/ContextApp/EventButtonPressActivity.java

```diff
@@ -0,0 +1,27 @@
+package com.github.migi_1.ContextApp;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+/**
+ *
+ * @author Nils
+ */
+public class EventButtonPressActivity extends Activity {
+        @Override
+        public void onCreate(Bundle savedInstanceState) {
+            super.onCreate(savedInstanceState);
+            setContentView(R.layout.android_event_button);
+            Button returnButton = (Button) findViewById(R.id.event1_button);
+            returnButton.setOnClickListener(new View.OnClickListener() {
+
+                @Override
+                public void onClick(View v) {
+                    finish();
+                }
+            });
+        }
+}
```

Comment:
This trigger button is implemented in the feature_randomEvent branch and is therefore not needed in this PR. This happened due to a merge to make sure the app loads correctly in an android app, at the time of the merge I was already working on that. Again, this is not needed in this PR, so everything that has to do with this trigger/event has to be removed from this PR.